frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -53,6 +53,7 @@ import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
 import { OverlayTriggerComponent } from './OverlayTriggerComponent';
 // eslint-disable-next-line import/no-unresolved
 import { diff } from 'deep-object-diff';
+import { format } from 'date-fns';
```

Comment:
Can we use `moment` instead? Looks like this package is not used anywhere in FE and we can remove it

Reply:
Done